### PR TITLE
Adding usage examples to docstrings in record models 

### LINKED ISSUE #415: [Python client] Add usage example to docstrings in record models
Include usage examples for:

Text2Text, TextClassification, TokenClassification records to be shown in the docs here:
https://rubrix.readthedocs.io/en/stable/reference/python_client.html#module-rubrix.client.models

You can see how it's done for the methods here for example:

https://github.com/recognai/rubrix/blob/02e6b752ed445acd076dcda7a910cfdb96bc05ba/src/rubrix/__init__.py#L99

The relevant file is src/client/models.py: https://github.com/recognai/rubrix/blob/master/src/rubrix/client/models.py

### PR DESCRIPTION
**Pull request by issam9**
Adding usage examples to Text2Text, TextClassification, TokenClassification records

### CODE REVIEW

File: src/rubrix/client/models.py [hunk 1]
```diff
@@ -88,6 +88,13 @@ class TextClassificationRecord(BaseModel):
             If an annotation is provided, this defaults to 'Validated', otherwise 'Default'.
         event_timestamp:
             The timestamp of the record.
+
+    Examples:
+        >>> import rubrix as rb
+        ... record = rb.TextClassificationRecord(
```
Comment (dcfidalgo) [inline src/rubrix/client/models.py]:
```suggestion
        >>> record = rb.TextClassificationRecord(
```

File: src/rubrix/client/models.py [hunk 2]
```diff
@@ -88,6 +88,13 @@ class TextClassificationRecord(BaseModel):
...
+        ...     inputs={"text": "my first rubrix example"},
+        ...     prediction=[('spam', 0.8), ('ham', 0.2)]
+        >>> )
```
Comment (dcfidalgo) [inline src/rubrix/client/models.py]:
```suggestion
        ... )
```

File: src/rubrix/client/models.py [hunk 3]
```diff
@@ -154,6 +161,14 @@ class TokenClassificationRecord(BaseModel):
             If an annotation is provided, this defaults to 'Validated', otherwise 'Default'.
         event_timestamp:
             The timestamp of the record.
+    
+    Examples:
+        >>> import rubrix as rb
+        ... record = rb.TokenClassificationRecord(
```
Comment (dcfidalgo) [inline src/rubrix/client/models.py]:
```suggestion
        >>> record = rb.TokenClassificationRecord(
```

File: src/rubrix/client/models.py [hunk 4]
```diff
@@ -154,6 +161,14 @@ class TokenClassificationRecord(BaseModel):
...
+        ...     text = "Michael is a professor at Harvard",
+        ...     tokens = ["Michael", "is", "a", "professor", "at", "Harvard"],
+        ...     prediction = [('NAME', 0, 7), ('LOC', 26, 33)]
+        >>> )
```
Comment (dcfidalgo) [inline src/rubrix/client/models.py]:
```suggestion
        ... )
```

File: src/rubrix/client/models.py [hunk 5]
```diff
@@ -207,6 +222,13 @@ class Text2TextRecord(BaseModel):
             If an annotation is provided, this defaults to 'Validated', otherwise 'Default'.
         event_timestamp:
             The timestamp of the record.
+    
+    Examples:
+        >>> import rubrix as rb
+        ... record = rb.Text2TextRecord(
```
Comment (dcfidalgo) [inline src/rubrix/client/models.py]:
```suggestion
        >>> record = rb.Text2TextRecord(
```

File: src/rubrix/client/models.py [hunk 6]
```diff
@@ -207,6 +222,13 @@ class Text2TextRecord(BaseModel):
...
+        ...     text="My name is Sarah and I love my dog.",
+        ...     prediction=["Je m'appelle Sarah et j'aime mon chien."]
+        >>> )
```
Comment (dcfidalgo) [inline src/rubrix/client/models.py]:
```suggestion
        ... )    
```

**Review by dcfidalgo — CHANGES_REQUESTED** [review]
Thanks a lot! Just some minor corrections regarding the [doctest format](https://docs.python.org/3/library/doctest.html). I noticed that the format is also wrong for our examples in the `rubrix.__init__` file. I will provide a PR for these.

**Comment by issam9** [comment]
Okay, thanks

**Review by dcfidalgo — APPROVED** [review]

**Comment by dcfidalgo** [comment]
LGTM, thanks a lot @issam9

**Pull request merged**